src/components/NoticeText/CampaignRegistration10DlcNoticeText.tsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+
+const CampaignRegistration10DlcNoticeText: React.FC = () => (
+  <>
+    <p>
+      <b>
+        You have started but not yet completed your 10DLC registration form.
+      </b>
+      <br /> <br />
```

Comment:
I would avoid using `<br>` line break elements to separate distinct bodies of text or to create margins. Instead, use `<p>` paragraph elements with styling. `<br>` elements are for single bodies of text where line breaks in the text have meaning (like poems or addresses).

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br
```suggestion
    </p>
    <p>
```

Reply:
yup good catch